neuralcoref/neuralcoref.pyx

```diff
@@ -20,6 +20,13 @@ import array
 from libc.stdint cimport uint16_t, uint32_t, uint64_t, uintptr_t, int32_t
 
 import numpy
+try:
+    import cupy
+    to_numpy = cupy.asnumpy
```

Comment:
Maybe we can avoid this dependency check by relying on Thinc for that.

Reply:
How do you feel about this?

```Python
def to_numpy(a):
    if thinc.neural.util.is_cupy_array(a):
        import cupy
        return cupy.asnumpy(a)
    else:
        return a
```

That way there isn't a conditional import, but we still have to `import cupy`. I'm not that familiar with thinc, but the thinc source does not use `cupy.asnumpy()` anywhere, so there probably isn't a good wrapper.